test/physics/base/PhysicsTestBase.hh

```diff
@@ -78,12 +84,18 @@ class PhysicsTestBase : public celeritas::Test
         return make_span(interactions_);
     }
 
+    //!@{
+    //! Random number generator
+    RandomEngine& rng() { return rng_; }
+    //!@}
+
   private:
     //// DATA ////
 
     SPConstMaterials materials_;
     SPConstParticles particles_;
     SPConstPhysics   physics_;
+    RandomEngine     rng_;
```

Comment:
@pcanal Sorry I just noticed this addition -- the `PhysicsTestBase` class is also used by device tests (which of course don't use the host RNG). Can you move this up to the PhysicsStepUtils harness, which is host-only?